src/GitHub.VisualStudio/GitHub.VisualStudio.imagemanifest

```diff
@@ -1,7 +1,10 @@
 ﻿<ImageManifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://schemas.microsoft.com/VisualStudio/ImageManifestSchema/2014">
 
   <Symbols>
+    <!-- We need to ensure that `GitHub.VisualStudio` has loaded before any of these resources are used. -->
+    <!-- This should happen when `AssemblyResolverPackage` auto-loads on `UICONTEXT.NoSolution`. -->
     <String Name="Resources" Value="/GitHub.VisualStudio;component/Resources/icons" />
```

Comment:
This seems the culprit was actually here. It tries to find an assembly by its simple name before that assembly has been loaded. I think when the `Team Explorer` related assemblies get initialized might have changed in VS 15.3. This can cause `GitHub.VisualStudio` not to get pre-loaded and the icons fail to load.

There is a possible workaround for this:
https://github.com/Microsoft/PTVS/commit/ec1b895f9410c4fa51d05dc62b47ae0ab854d3e5

I did a spike to see if this might work for us, but it would involve moving a bunch of resources into `GitHub.UI` and fixing a bunch of other issues (not a point fix). This might be worth investigating at some future time.